gulpfile.js

```diff
@@ -0,0 +1,152 @@
+import url from 'url';
+import del from 'del';
+import gulp from 'gulp';
+import log from 'fancy-log';
+import mkdirp from 'mkdirp';
+import selfsigned from 'selfsigned';
+import { promises as fs } from 'fs';
+import Graceful from 'node-graceful';
+import { spawn } from 'child_process';
+import { Docker } from 'docker-cli-js';
+import { cleanEnv, str } from 'envalid';
+import passportSaml from 'passport-saml';
+
+const SamlStrategy = passportSaml.Strategy;
+
+const TEST_WEBSITE_TUNNEL_IMAGE = 'educandu/inlets:1.0.0';
+const TEST_WEBSITE_TUNNEL_CONTAINER_NAME = 'website-tunnel';
+
+let server = null;
+Graceful.on('exit', () => {
+  server?.kill();
+});
+
+const isMac = process.platform === 'darwin';
+const containerCommandTimeoutMs = isMac ? 2000 : 1000;
```

Comment:
👍

Reply:
We should maybe also test this branch on one of the macs before merging.

Reply:
Sure, let's do that together 🙏